test/navigation/preprocess_encoders.cpp

```diff
@@ -0,0 +1,68 @@
+#include <cmath>
+
+#include <iostream>
+
+#include <gtest/gtest.h>
+
+#include <core/logger.hpp>
+#include <core/types.hpp>
+#include <navigation/preprocess_encoders.hpp>
+#include <utils/manual_time.hpp>
+
+namespace hyped::test {
+
+core::Float epsilon = 1e-5;

```

Comment:
`static constexpr core::Float kEpsilon = ...`

move this into a header such as `test/navigation/utils.hpp` and have other files include it, as required

Reply:
Isn't this already in core::types::kEpsilon?